api/src/main/java/org/pmiops/workbench/db/model/CohortReview.java

```diff
@@ -0,0 +1,193 @@
+package org.pmiops.workbench.db.model;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "cohort_review")
+public class CohortReview {
+
+    private long cohortReviewId;
+    private long cohortId;
+    private long cdrVersionId;
+    private Timestamp creationTime;
+    private Timestamp lastModifiedTime;
+    private long matchedParticipantCount;
+    private long reviewedCount;
```

Comment:
Looks like you're not setting or getting this below. 

I could go either way on whether we cache this... but if we're not going to, remove the field.